llvm/test/CodeGen/AIE/aie2/internalize.ll

```diff
@@ -0,0 +1,58 @@
+; RUN: opt -O1 -S -mtriple=aie2-none-unknown-elf -aie2-internalize-symbols -aie2-internalize-skip-function="_main" < %s | FileCheck %s
```

Comment:
I think we can have an additional test considering two function in the list.

Reply:
Added..